app/assets/javascripts/application.js

```diff
@@ -10,7 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
+= require jquery
```

Comment:
That // is important. I have no idea what it does without it. //= is the syntax for including something